src/gamestate/dcon_generated.txt

```diff
@@ -3761,6 +3766,11 @@ object {
 		type{ array{commodity_id}{float} }
 		tag{ save }
 	}
+	property {
```

Comment:
This should be removed, as it isn't part of this PR

Reply:
Fixed w #13ae911